rita/src/rita_common/tunnel_manager/mod.rs

```diff
@@ -281,9 +282,8 @@ impl Handler<PortCallback> for TunnelManager {
 }
 
 pub fn make_babel_stream() -> Result<TcpStream, Error> {
-    let stream = TcpStream::connect::<SocketAddr>(
-        format!("[::1]:{}", SETTING.get_network().babel_port).parse()?,
-    )?;
+    let stream = open_babel_stream(SETTING.get_network().babel_port)?;
```

Comment:
a lot of these babel changes shouldn't be here, did you pull in my recent babel pull yourself somehow?